Sources/Pathos/pathAnalysis.swift

```diff
@@ -64,7 +63,7 @@ public func normalize(path: String) -> String {
 public func splitDrive(path: String) -> (String, String) {
 #if os(Windows)
     let seq = path.map { $0 == "/" ? "\\" : $0 }
-    if seq.count > 2 && seq.starts(with: #"\\"#) && seq[2] != "\\" {
+    if seq.starts(with: #"\\"#) && seq[2] != "\\" {
```

Comment:
this could crash if `seq` is `\\` because `seq[2]` goes out of bound.

Reply:
oops, I somehow repeatedly saw the `>` as `>=`. 🤦‍♂️